web/templates/precall.ejs

```diff
@@ -24,9 +24,21 @@
   <form method="post">
     <div id="video-preview">
     </div>
+    <% if (!autoGenerateRoomName) { %>
+    <div id="enter-room-name-prompt">
+      <div class="room-name-input-container">
+        <input id="room-name-input" class="block username" name="roomName" autocomplete="off" maxlength="46" value="<%=roomName%>">
+        <label>Room Name</label>
+        <p class="error-room error-text">
+          <i data-icon="warning"></i>
+          Please enter a meeting name.
+        </p>
+      </div>
+    </div>
+    <% } %>
     <div id="enter-name-prompt">
       <div class="user-name-input-container">
-        <input id="user-name-input" class="block username" name="userName" autocomplete="off" data-wd="username" autofocus maxlength="46">
+        <input id="user-name-input" class="block username" name="userName" autocomplete="off" data-wd="username" maxlength="46">
```

Comment:
is it good idea to validate that username is always present when the auto-generate is turned off ?

```suggestion
        <input id="user-name-input" class="block username" name="userName" autocomplete="off" data-wd="username" maxlength="46" required>
```

Reply:
We explicitly allow for blank usernames. They are displayed as "Anonymous user" in the room view

Reply:
alright then it looks fine